polyfill.js

```diff
@@ -0,0 +1,137 @@
+'use strict'
+
+if (typeof Symbol !== 'function' || !Symbol.isConcatSpreadable) {
```

Comment:
True, but for this specific use case it doesn't _need_ to be the real Symbol for the code to execute correctly.